otelcollector/shared/tomlparser-ccp-default-targets-metrics-keep-list.go

```diff
@@ -102,18 +103,22 @@ func populateSettingValuesFromConfigMap(parsedConfig map[string]interface{}) (Re
 	}
 
 	// Logging the values being set
-	fmt.Printf("controlplaneKubeControllerManagerRegex: %s\n", regexValues.ControlplaneKubeControllerManager)
-	fmt.Printf("controlplaneKubeSchedulerRegex: %s\n", regexValues.ControlplaneKubeScheduler)
-	fmt.Printf("controlplaneApiserverRegex: %s\n", regexValues.ControlplaneApiserver)
-	fmt.Printf("controlplaneClusterAutoscalerRegex: %s\n", regexValues.ControlplaneClusterAutoscaler)
-	fmt.Printf("controlplaneEtcdRegex: %s\n", regexValues.ControlplaneEtcd)
-	fmt.Printf("minimalIngestionProfile: %s\n", regexValues.MinimalIngestionProfile)
+	fmt.Printf("populateSettingValuesFromConfigMap::controlplaneKubeControllerManagerRegex: %s\n", regexValues.ControlplaneKubeControllerManager)
+	fmt.Printf("populateSettingValuesFromConfigMap::controlplaneKubeSchedulerRegex: %s\n", regexValues.ControlplaneKubeScheduler)
+	fmt.Printf("populateSettingValuesFromConfigMap::controlplaneApiserverRegex: %s\n", regexValues.ControlplaneApiserver)
+	fmt.Printf("populateSettingValuesFromConfigMap::controlplaneClusterAutoscalerRegex: %s\n", regexValues.ControlplaneClusterAutoscaler)
+	fmt.Printf("populateSettingValuesFromConfigMap::controlplaneEtcdRegex: %s\n", regexValues.ControlplaneEtcd)
+	fmt.Printf("populateSettingValuesFromConfigMap::minimalIngestionProfile: %s\n", regexValues.MinimalIngestionProfile)
+
+	if regexValues.MinimalIngestionProfile == "false" {
+		minimalIngestionProfile = "false"
+	}
 
 	return regexValues, nil // Return regex values and nil error if everything is valid
 }
 
 func populateRegexValuesWithMinimalIngestionProfile(regexValues RegexValues) {
-	if regexValues.MinimalIngestionProfile == "true" {
+	if minimalIngestionProfile == "true" {
```

Comment:
if its false, don't we want to do the same thing as true ?

Reply:
Ok, so collect minimalingestion profile metrics always. The setting is becomes redundant for CCP metrics then? I'll create a task to update the docs too for when this release rolls out.

Reply:
Pushed in an update for this.